Automated Smoketest: :x:
Sha: 84e0a28b87f1341d7d6c724ad2a4ea9db0f1e8a0
Build: failed
Number of Error(s): 42
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:821: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dali.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:844: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dali.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:864: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dali.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:897: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/daliplugin.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1049: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dafilesrv.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1097: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dfuserver.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1115: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dfuserver.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1295: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/dropzone.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1413: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/eclagent_config.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1431: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/eclagent_config.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1490: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/eclccserver.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1560: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/eclscheduler.xsd.mod.xml, and no fallback was found
/mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/UsingConfigManager/UsingConfigManager.xml:1618: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-10527/HPCC-Platform/docs/XMLGeneration/xml/esp.xsd.mod.xml, and no
 ... ( comment is too long, 49 lines, 10877 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

--- a/docs/ECLStandardLibraryReference/SLR-Mods/GetEspUrl.xml
+++ b/docs/ECLStandardLibraryReference/SLR-Mods/GetEspUrl.xml
@@ -1,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="GetEspUrl">
+  <title><emphasis role="bold">GetEspUrl</emphasis></title>
+
+  <para><emphasis>result</emphasis> <emphasis role="bold"> :=
+  STD.File.GetEspUrl <indexterm>
+      <primary>STD.File.GetEspUrl</primary>
+    </indexterm><indexterm>
+      <primary>File.GetEspUrl</primary>
+    </indexterm> <indexterm>
+      <primary>GetEspUrl</primary>
+    </indexterm>( [</emphasis> <emphasis>username, userPW </emphasis>
+  <emphasis role="bold">]);</emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>username</emphasis></entry>
+
+          <entry>Optional. A STRING containing a username to use for
+          authenticated access to the ESP process. If omitted, it indicates
+          that no user authentication is required. </entry>
+        </row>
+
+        <row>
+          <entry><emphasis>userPW</emphasis></entry>
+
+          <entry>Optional. A STRING containing the password to use with the
+          user cited in the <emphasis>username</emphasis> argument. If
+          <emphasis>username</emphasis> is empty then this is ignored</entry>
+        </row>
+
+        <row>
+          <entry>Return:<emphasis> </emphasis></entry>
+
+          <entry>GetEspUrl returns a STRING containing the full URL (including
+          HTTP scheme and port) to an ESP server process. If more than one ESP
+          process is defined then the first found process is returned. Returns
+          an empty string if an ESP server process cannot be found in the
+          environment.</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>The <emphasis role="bold">GetEspUrl </emphasis>function returns the
+  full URL to an ESP server process. </para>
+
+  <para>Example:</para>
+
+  <programlisting format="linespecific">EspAddress := STD.File.GetEspUrl
+</programlisting>
+</sect1>

